src/gazebo_mavlink_interface.cpp

```diff
@@ -1229,7 +1229,7 @@ void GazeboMavlinkInterface::VisionCallback(OdomPtr& odom_message) {
     odom.time_usec = odom_message->time_usec();
 
     odom.frame_id = MAV_FRAME_LOCAL_NED;
-    odom.child_frame_id = MAV_FRAME_BODY_FRD;
```

Comment:
The issue is here: The reason the frame got reserved is that it got stripped from MAVLink. So instead of changing the enum, the whole point is to change this for good.

Reply:
Do you mean by not using the enum at all?

Reply:
@Jaeyoung-Lim on the enum you can see this was deprecated and was replaced by another value.

Reply:
@Jaeyoung-Lim supposedly this was replaced by MAV_FRAME_BODY_OFFSET_NED -https://mavlink.io/en/messages/common.html#MAV_FRAME_RESERVED_12.